src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+
+```bash
+$ kubectl config use-context cluster2
+Switched to context "cluster2".
+```
+
+```bash
+$ kubectl -n submariner-operator get pods --watch
+NAME                                  READY   STATUS    RESTARTS   AGE
+submariner-operator-dcbdf5669-n7jgp   1/1     Running   0          12s
+submariner-gateway-btzrq              0/1     Pending   0          0s
+submariner-gateway-btzrq              0/1     Pending   0          0s
+submariner-gateway-btzrq              0/1     ContainerCreating   0          0s
+submariner-routeagent-bmgbc           0/1     Pending             0          0s
+submariner-routeagent-rl9nh           0/1     Pending             0          0s
+submariner-routeagent-bmgbc           0/1     Pending             0          0s
+submariner-routeagent-wqmzs           0/1     Pending             0          0s
+submariner-routeagent-rl9nh           0/1     Pending             0          0s
+submariner-routeagent-wqmzs           0/1     Pending             0          0s
+submariner-routeagent-rl9nh           0/1     ContainerCreating   0          0s
+submariner-routeagent-bmgbc           0/1     ContainerCreating   0          0s
+submariner-routeagent-wqmzs           0/1     ContainerCreating   0          0s
+submariner-lighthouse-agent-586cf4899-wn747   0/1     Pending             0          0s
+submariner-lighthouse-agent-586cf4899-wn747   0/1     Pending             0          0s
+submariner-lighthouse-agent-586cf4899-wn747   0/1     ContainerCreating   0          0s
+submariner-lighthouse-coredns-c88f64f5-qlw4x   0/1     Pending             0          0s
+submariner-lighthouse-coredns-c88f64f5-h77kw   0/1     Pending             0          0s
+```
+
+It may take a couple of minutes for all components to come up:
+
+```bash
+$ kubectl -n submariner-operator get pods
+NAME                                           READY   STATUS    RESTARTS   AGE
+submariner-gateway-btzrq                       1/1     Running   0          76s
+submariner-lighthouse-agent-586cf4899-wn747    1/1     Running   0          75s
+submariner-lighthouse-coredns-c88f64f5-h77kw   1/1     Running   0          73s
+submariner-lighthouse-coredns-c88f64f5-qlw4x   1/1     Running   0          73s
+submariner-operator-dcbdf5669-n7jgp            1/1     Running   0          89s
+submariner-routeagent-bmgbc                    1/1     Running   0          75s
+submariner-routeagent-rl9nh                    1/1     Running   0          75s
+submariner-routeagent-wqmzs                    1/1     Running   0          75s
+```
+
+You can use this command to verify on which node the Gateway Engine is running on:
+
+```bash
+$ kubectl get node --selector=submariner.io/gateway=true -o wide
+NAME              STATUS   ROLES    AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE       KERNEL-VERSION           CONTAINER-RUNTIME
+cluster2-worker   Ready    <none>   6h59m   v1.17.0   172.17.0.7    <none>        Ubuntu 19.10   5.8.18-200.fc32.x86_64   containerd://1.3.2
```

Comment:
Here the external-ip is shown as empty. 

On an AWS Cluster, I see the following.
```
[sgaddam@localhost vmware]$ kubectl get node --selector=submariner.io/gateway=true -o wide
NAME                          STATUS   ROLES    AGE   VERSION           INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                                                       KERNEL-VERSION                 CONTAINER-RUNTIME
ip-10-0-44-234.ec2.internal   Ready    worker   56d   v1.18.3+47c0e71   10.0.44.234   3.81.125.62   Red Hat Enterprise Linux CoreOS 45.82.202009181447-0 (Ootpa)   4.18.0-193.23.1.el8_2.x86_64   cri-o://1.18.3-15.rhaos4.5.gitae4ef7b.el8
```

Similarly on a VMWare OnPrem cluster
```
[sgaddam@localhost vmware]$  kubectl get node --selector=submariner.io/gateway=true -o wide
NAME                              STATUS   ROLES    AGE   VERSION           INTERNAL-IP    EXTERNAL-IP    OS-IMAGE                                                       KERNEL-VERSION                 CONTAINER-RUNTIME
hivemind-dev-dkghj-worker-86nns   Ready    worker   41d   v1.18.3+6c42de8   10.1.158.222   10.1.158.222   Red Hat Enterprise Linux CoreOS 45.82.202008290529-0 (Ootpa)   4.18.0-193.14.3.el8_2.x86_64   cri-o://1.18.3-11.rhaos4.5.gite5bcc71.el8
```